test-packages/memory-tests/sdk-canary/test-script.js

```diff
@@ -0,0 +1,115 @@
+const memwatch = require('node-memwatch-new');
+const { writeFileSync } = require('fs');
+const { resolve } = require('path');
+const { testService } = require('./test-service');
+
+const dest = { url: 'http://localhost:4004/' };
+
+const test = async () => {
+  // The script is written in js (not ts) and executed with --inspect flag because of this lib.
+  const hd = new memwatch.HeapDiff();
+
+  const testEntityApi = testService().testEntityApi;
+  const testEntities101 = await testEntityApi
+    .requestBuilder()
+    .getAll()
+    .execute(dest);
+  const testEntities102 = await testEntityApi
+    .requestBuilder()
+    .getAll()
+    .execute(dest);
+  const testEntities103 = await testEntityApi
+    .requestBuilder()
+    .getAll()
+    .execute(dest);
+  const testEntities104 = await testEntityApi
+    .requestBuilder()
+    .getAll()
+    .execute(dest);
+  const testEntities105 = await testEntityApi
+    .requestBuilder()
+    .getAll()
+    .execute(dest);
+  const testEntities106 = await testEntityApi
+    .requestBuilder()
+    .getAll()
+    .execute(dest);
+  const testEntities107 = await testEntityApi
```

Comment:
Don't you like for loops? (or for async probably `.reduce`)

Reply:
Here I would like to keep the new entities in the memory.
But you are right, I'll save them in an array in a loop.